src/test/java/teammates/test/cases/ui/BaseActionTest.java

```diff
@@ -324,7 +324,7 @@ protected void verifyAccessibleForStudentsOfTheSameCourse(String[] submissionPar
         StudentAttributes student1InCourse1 = data.students.get("student1InCourse1");
         
         gaeSimulation.loginAsStudent(student1InCourse1.googleId);
-        verifyCanAccess(submissionParams);
+        verifyCanAccess(addUserIdToParams(student1InCourse1.googleId, submissionParams));
```

Comment:
@kanghj We are using the parameter `user` even if the user is logged in as student or instructor.

For student
![screenshot from 2016-07-26 14 23 29](https://cloud.githubusercontent.com/assets/12219700/17131780/a6fcdee2-533c-11e6-9fcc-10877aacd933.png)

For instructor
![screenshot from 2016-07-26 14 28 12](https://cloud.githubusercontent.com/assets/12219700/17132738/b17460d0-5340-11e6-9a51-04bb6cdc1b15.png)

I am using `user` parameter, for sending email. therefore it should be not null now.
Getting following failed test on removing this.
`java.lang.AssertionError: The supplied parameter was null

	at teammates.common.util.Assumption.fail(Assumption.java:61)
	at teammates.common.util.Assumption.assertTrue(Assumption.java:29)
	at teammates.common.util.Assumption.assertFalse(Assumption.java:46)
	at teammates.logic.api.Logic.sendConfirmationEmailForSubmission(Logic.java:1247)
	at teammates.ui.controller.FeedbackSubmissionEditSaveAction.execute(FeedbackSubmissionEditSaveAction.java:181)
	at teammates.ui.controller.Action.executeAndPostProcess(Action.java:344)
	at teammates.test.cases.ui.BaseActionTest.verifyCanAccess(BaseActionTest.java:629)
	at teammates.test.cases.ui.BaseActionTest.verifyAccessibleForStudentsOfTheSameCourse(BaseActionTest.java:332)
	at teammates.test.cases.ui.BaseActionTest.verifyOnlyStudentsOfTheSameCourseCanAccess(BaseActionTest.java:291)
	at teammates.test.cases.ui.AllActionsAccessControlTest.testStudentFeedbackSubmissionEditSave(AllActionsAccessControlTest.java:1034)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
`